### PR TITLE
enhancements/installer/component-selection: Clarify cluster-bot testing

### PR DESCRIPTION
**Pull request by wking**
And broken-test-case ownership.

### CODE REVIEW

File: enhancements/installer/component-selection.md [hunk 1]
```diff
@@ -358,27 +358,49 @@ Our current plan is to only annotate resources that are not depended upon (aka "
...
+##### No-capabilities testing
+
+Install a cluster with the various add-on components excluded and confirm that the cluster is functional but only running the expected add-ons.
+This will necessitate disabling or modifying tests that depend on the disabled component(s).
+
+When introducing a new capability would introduce test-case failures, it is the responsibility of the team driving the new capability to address the issue.
+For example, they may [teach an origin test-case to reduce expectations in the face of a disabled capability][bump-origin].
+Or they may [adjust][bump-auth-1] [another component][bump-auth-2] to help it gracefully handle the new capability's absence.
+
+When a component change or test-suite change breaks this test, it is the responsibility of the team that made the breaking change to address the issue.
```
Comment (bparees) [inline enhancements/installer/component-selection.md:377]:
```suggestion
If a future a component change or test-suite change breaks this test(i.e. introduces or changes a test such that testing on no-capabilities clusters now fails), it is the responsibility of the team that made the breaking change to address the issue.
```

Comment (wking) [reply enhancements/installer/component-selection.md]: picked up with bd37a642 -> be5dff1b, which also adds a space before your opening paren.

**Pull request merged**